syfertext/pipeline/simple_tagger.py

```diff
@@ -142,6 +150,7 @@ def _get_tag(self, token: Token):
         return tag
 
     @staticmethod
+    # Find return type: Tuple of what?
     def simplify(worker: BaseWorker, simple_tagger: "SimpleTagger"):
```

Comment:
tuple of str.Union[set, list, dict],object,bool

Reply:
I did not get that completely